model/facade/action/ExtraHoursReportAction.php

```diff
@@ -153,145 +152,125 @@ private function numWorkDays(DateTime $init, DateTime $end) {
      * @param DateTime $end the ending date of the interval.
      * @param UserVO $user the User whose extra hours we want to compute.
      */
-    private function netExtraHours(DateTime $init, DateTime $end, UserVO $user = NULL) {
+    private function netExtraHours(DateTime $init, DateTime $end, UserVO $userVO) {
+
+        if (is_null($userVO->getId()))
+            return array();
 
         $taskDao = DAOFactory::getTaskDAO();
         $commonDao = DAOFactory::getCommonEventDAO();
         $journeyHistoryDao = DAOFactory::getJourneyHistoryDAO();
         $cityHistoryDao = DAOFactory::getCityHistoryDAO();
 
-        if (is_null($user))
-        {
-            $groupDAO = DAOFactory::getUserGroupDAO();
-            $users = $groupDAO->getUsersByUserGroupName(
-                    ConfigurationParametersManager::getParameter("ALL_USERS_GROUP"));
-        }
-        else
-        {
-            $users = array($user);
-        }
-
         $userWork = array();
 
-        foreach($users as $userVO)
-        {
-            if (!is_null($userVO->getId()))
-            {
-                $hoursWorked = $taskDao->
-                        getTaskReport($userVO, $init, $end, "USER");
+        $hoursWorked = $taskDao->
+                getTaskReport($userVO, $init, $end, "USER");
 
```

Comment:
Note for reviewer: the chunk of changes starting here is just re-indentation, no actual code was modified.